BSP/STM32F413/Makefile

```diff
@@ -12,7 +14,11 @@ READABLE = true
 # optimization
 # change to -Og if you want the compiler to generate assembly that is very close
 # to your source code. Change to -O3 for best performance
+ifneq ($(debug), 0)
```

Comment:
also maybe make this ifeq ($(debug) instead of ifneq ($(debug)) and reverse the order of the clauses. this doesn't actually matter it's just a personal preference ig.